src/main/java/gregtech/common/metatileentities/multi/multiblockpart/MetaTileEntityFluidHatch.java

```diff
@@ -56,12 +65,14 @@ public MetaTileEntity createMetaTileEntity(IGregTechTileEntity tileEntity) {
     @Override
     public NBTTagCompound writeToNBT(NBTTagCompound data) {
         super.writeToNBT(data);
+        data.setBoolean("workingEnabled", workingEnabled);
         return data;
     }
 
     @Override
     public void readFromNBT(NBTTagCompound data) {
         super.readFromNBT(data);
+        this.workingEnabled = data.getBoolean("workingEnabled");
```

Comment:
This should check `data.hasKey("WorkingEnabled")`, as the current MBParts will not have this key.

Basically
```java
if(data.hasKey("WorkingEnabled") {
    this.workingEnabled = data.getBoolean('workingEnabled");
}
else {
    this.workingEnabled = true;
}
````

Reply:
The constructor already sets it to true be default